<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCMESHTRANGUI_TranslateMeshDlg</class>
 <widget class="QDialog" name="MCMESHTRANGUI_TranslateMeshDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Move and Rotate </string>
  </property>
  <widget class="QGroupBox" name="tr_gb_mv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Move Vector</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="tr_le_dx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tr_le_dy">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tr_le_dz">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dz</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh </string>
   </property>
   <widget class="QComboBox" name="tr_cb_mt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>------</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesh </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tr_le_nmt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tr_cb_gp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>------</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tr_pb_gpsl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
